internal/search/alert/observer.go

```diff
@@ -210,7 +210,10 @@ func (o *Observer) Done() (*search.Alert, error) {
 
 type alertKind string
 
-const luckySearchQueries alertKind = "lucky-search-queries"
+const (
+	smartSearchAdditionalResults alertKind = "smart-search-additional-results"
+	smartSearchPureResults                 = "smart-search-pure-results"
```

Comment:
Should this line also have `alertKind`?

```suggestion
	smartSearchPureResults alertKind = "smart-search-pure-results"
```